### PR TITLE
Improvements to the vfc_precexp exploration script

### LINKED ISSUE #208: vfc_precexp hardcodes options
Some important search options are hardcoded in the vfc_precexp script.
For example, the VPREC mode is set to all and there is no way of changing it.
The search script also goes through all possible modes of --funct-inst (operations, arguments and all) and there is no way to choose which exploration to do.
For some cases, it is useful to get faster results by exploring just one desired mode.

### PR DESCRIPTION
**Pull request by mistoan**
This pull request fixes the issues with the vfc_precexp script mentioned in issues #208, #211 and #213.
Fixes #208, #211 and #213

The script can now accept the following options:
 - the execution script
 - the compare script
 - the maximum execution time each run
 - the mode for inst-func
 - the mode for vprec

I've also added checks in the configuration file for the required Python modules, as this seemed to be missing.

### CODE REVIEW

File: src/vfc_precexp [hunk 1]
```diff
@@ -350,24 +356,49 @@ def search_minimum_arguments(	runPath,
...
+        print("explore.py [runPath] [cmpPath] functions ...")
         exit()
 
-    runPath = sys.argv[1]
-    cmpPath = sys.argv[2]
-    function_list = []
-
-    for i in range(3, len(sys.argv)):
-        function_list.append(sys.argv[i])
+    # runPath = sys.argv[1]
```
Comment (pablooliveira) [inline src/vfc_precexp:379]: Please remove these commented lines.

**Review by pablooliveira — APPROVED** [review]
LGTM, please remove unneeded comment before merging.

File: src/vfc_precexp (hunk 1 above)
Comment (mistoan) [reply src/vfc_precexp]: I realized this issue just before you replied and had already committed the cleaned-up version.

Comment (pablooliveira) [reply src/vfc_precexp]: Excellent. I let you merge then.

**Comment by mistoan** [comment]
Great!
I've merged the changes.

**Pull request merged**